### PR TITLE
Added search functionality. Search endpoint is /api/search/?name=SearchSomething

### PR DESCRIPTION
**Pull request by Nirav-1999**

### CODE REVIEW

**Comment by pujanm** [comment]
Please check the build error.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:\\Users\\acer\\.virtualenvs\\BE-Project-Web-JO3fbHpy\\Scripts\\python.exe"
+}
```
[the remark this answers is not in the document]
Comment (Rashmil-1999) [reply .vscode/settings.json]: He might not have that updated file

**Comment by pujanm** [comment]
Please don't change the `.gitignore` file it was proper. Because other useless files are getting in. Otherwise, your work looks good to me and is mergeable.

**Comment by Nirav-1999** [comment]
I didn't make any changes to the gitignore.
It was the same as what Rashmil pushed.
Then too I will check it again Tomorrow.

**Comment by pujanm** [comment]
I guess everything was proper till the second last commit, your latest commit did this.
Check it here: https://github.com/djunicode/BE-Project-Web/pull/26/commits

**Comment by Rashmil-1999** [comment]
I saw the django rest-framework doc and found out that there is no need to write more views we can simply use our existing views to make queries to it and get the required data
I'll do the required changes and make a new PR If that is okay.
Please Acknowledge @pujanm  @Nirav-1999

**Comment by pujanm** [comment]
@Rashmil-1999 if it simplifies the code then yes.

**Comment by pujanm** [comment]
Closing this PR since @Rashmil-1999 has updated more changes over this one.

**Pull request closed without merging**